source/solvers/navier_stokes_base.cc

```diff
@@ -1424,13 +1437,13 @@ NavierStokesBase<dim, VectorType, DofsType>::postprocess_fd(bool firstIter)
             simulation_parameters.post_processing.flow_rate_output_name +
             ".dat";
           std::ofstream output(filename.c_str());
-          flow_rate_table.set_precision("time", 12);
+          flow_rate_table.set_precision("time", 4);
           for (unsigned int boundary_id = 0;
                boundary_id < simulation_parameters.boundary_conditions.size;
                ++boundary_id)
             flow_rate_table.set_precision("flow-rate-" +
                                             std::to_string(boundary_id),
```

Comment:
Use Utilities::int_to_string instead
and set the padding manually to 2, because we will never have more than 99 boundary conditions (it makes no sense)